ani-cli

```diff
@@ -251,32 +252,35 @@ menu_line_strong() {
 #################
 
 anime_selection () {
-	# user input for selecting anime
-	count=1
-	while read -r anime_id; do
-		menu_line_alternate "$anime_id" "$count"
-		: $((count+=1))
-	done <<-EOF
-	$search_results
-	EOF
-
-	prompt "Enter number"
-	choice="$REPLY $REPLY2"
-
-	# Check if input is a number
-	[ "$choice" -eq "$choice" ] 2>/dev/null || die "Invalid number entered"
-
-	# Select respective anime_id
-	count=1
-	while read -r anime_id; do
-		if [ "$count" -eq "$choice" ]; then
-			selection_id=$anime_id
-			break
-		fi
-		count=$((count+1))
-	done <<-EOF
-	$search_results
-	EOF
+	if [ -n "$ep_choice_to_start" ] && [ -n "$select_first" ]; then
+		selection_id="$(echo $search_results | sed 's/\ .*//' )"
```

Comment:
- [ ] Don't echo pipe into sed
- [ ] Is it really impossible to put $search_results in quotes?

Reply:
Cannot do either cause it would break.

Reply:
Fixed it by using a different method